projects/laji/src/app/shared-modules/spreadsheet/service/spreadsheet.service.ts

```diff
@@ -268,17 +269,19 @@ export class SpreadsheetService {
   }
 
   private getCombinedCoordinateValue(values: {[key: string]: string}): string {
-    if (!values[GeneratorService.splitCoordinate.system]) {
+    const system = values[GeneratorService.splitCoordinate.system] || values[GeneratorService.splitCoordinate.sys];
+
+    if (!system) {
       return values[GeneratorService.splitCoordinate.N] + ' ' + values[GeneratorService.splitCoordinate.E];
     }
     const suffix = typeof values[GeneratorService.splitCoordinate.N] === 'undefined' ||
       typeof values[GeneratorService.splitCoordinate.E] === 'undefined' ?
-      ' ' + values[GeneratorService.splitCoordinate.system] : '';
+      ' ' + system : '';
```

Comment:
The ternary operation is a bit hard to read, I though the string would end up having "system :"

How about:

```
const suffix = (
  typeof values[GeneratorService.splitCoordinate.N] === 'undefined'
  || typeof values[GeneratorService.splitCoordinate.E] === 'undefined'
) ? ' ' + system
  : '';
```